includes/rest-api/Controllers/Version2/class-wc-rest-system-status-v2-controller.php

```diff
@@ -1172,22 +1172,27 @@ public function get_pages() {
 			_x( 'Shop base', 'Page setting', 'woocommerce' ) => array(
 				'option'    => 'woocommerce_shop_page_id',
 				'shortcode' => '',
+				'block'     => '',
 			),
 			_x( 'Cart', 'Page setting', 'woocommerce' ) => array(
 				'option'    => 'woocommerce_cart_page_id',
 				'shortcode' => '[' . apply_filters( 'woocommerce_cart_shortcode_tag', 'woocommerce_cart' ) . ']',
+				'block'     => apply_filters( 'woocommerce_cart_block_tag', 'woocommerce/cart' ),
```

Comment:
Why is this (or for that matter shortcode too... but that's already a done deal) filtered? does it need to be filtered?

Reply:
This allows 3rd party plugins to provide valid blocks that can act as cart or checkout. If we don't filter it then `woocommerce/cart` becomes the only valid block here.

Reply:
Rationale sounds fair, I just don't think we should expose this filter until someone demonstrates why they'd want it. Not a blocker imo though, but probably should be weighed in on by @woocommerce/proton.